docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       - ./nginx/etc/nginx/conf.d/default.conf:/etc/nginx/conf.d/default.conf
 
   app-frontend:
-    image: node:4.4.7-wheezy
```

Comment:
The last time I tried Node v6 on a project, a bunch of the modules failed to install/work. I also think that Node v4 is the LTS, so I'd lean toward sticking with that if there isn't a compelling reason for use v6.

See: https://github.com/nodejs/LTS